executor/README.md

```diff
@@ -0,0 +1,65 @@
+# Seldon Executor
+
+This Go project replaces the Seldon Java Engine.  It is presently in development.
+
+** Do not use in production **
+
+
+## Functionality
+
+The focus is to provide a smaller more efficient graph orchestror. It itends t provide:
+
+ * http REST Seldon protocol server for Seldon graphs.
+ * grpc Seldon protocol server for Seldon graphs.
+ * Ability to handle other protocols in future, e.g. Tensorflow or NVIDIA TensorRT Server.
```

Comment:
Would tracing still be supported by executor?

Reply:
Yes. I think this should be supported.